### ISSUE TITLE
Fix the service worker

### ISSUE THREAD

**Issue by GavinJoyce** [glimmerjs/glimmer-api-docs#28]
TODO:

 * [x] Temporarily disable the service worker (as being able to access the docs is more important than raw load time): https://github.com/glimmerjs/glimmer-api-docs/pull/27
 * [ ] Fix the service worker (perhaps we need some ability to bust the cache?)

---

It seems that the current service worker can result in empty docs pages for some people. Here's what I see on https://glimmerjs.com/api/:

<img width="1440" alt="screen shot 2017-11-05 at 07 50 01" src="https://user-images.githubusercontent.com/2526/32412952-f9642aa6-c1fd-11e7-834b-1032dee3810f.png">

Also reported here:

<img width="1220" alt="screen shot 2017-11-05 at 07 42 34" src="https://user-images.githubusercontent.com/2526/32412953-fe5b3bc6-c1fd-11e7-8dc3-5d4562703219.png">

**Comment by marcoow** [comment]
I just realized we're not fingerprinting assets so that https://github.com/glimmerjs/glimmer-api-docs/blob/master/workers/service.ts#L42 makes no sense of course 🤦‍♂️ 

I guess we should be fingerprinting though or is there any particular reason not to do it?

**Comment by rwjblue** [comment]
Ya, I think we should add fingerprinting back in.  It should be as simple as adding `broccoli-asset-rev` I think?

**Comment by kmoe** [comment]
I also get an empty docs page:

![good_start](https://user-images.githubusercontent.com/5575356/32548516-d0da7112-c47d-11e7-8f2d-44e971fef68d.png)

Please could you let me know if there is a workaround or some plaintext version of the API docs available. `docs-source/main.json` is quite hard to read.

**Comment by marcoow** [comment]
The fix for now is to delete the service worker manually (or use Safari ;) )

**Comment by t-sauer** [comment]
Even after deleting the service worker (or when using Safari) it currently doesn't work since one of the broken `main.json` files is deployed.

**Comment by kmoe** [comment]
Likewise, the service worker makes no difference (and I don't have Safari as I'm on Linux). I would recommend making your API docs app as simple as possible (render the JSON to HTML, serve it) to ensure high availability of essential developer content.

**Comment by marcoow** [comment]
I agree with you @kmoe but I think the idea was/is to use the API docs as a show case for Glimmer which makes some sense as well (although it has massively backfired already - mostly my fault messing up the service worker).
